miasm2/analysis/depgraph.py

```diff
@@ -195,6 +198,8 @@ class DependencyResult(DependencyState):
     """Container and methods for DependencyGraph results"""
 
     def __init__(self, ircfg, initial_state, state, inputs):
+
+        super(DependencyResult, self).__init__(state.loc_key, state.pending)
```

Comment:
`self.links = state.links` could already be done in the init and then removed from the current init, together with `_graph` init